README.md

```diff
@@ -215,34 +205,23 @@ niceUser
 
 ## Information Gathering
 
-The `aeacus` binary supports gathering information on Windows in cases where it's tough to gather what the scoring system can see.
+The `aeacus` binary supports gathering information (on **Windows** only) in cases where it's tough to gather what the scoring system can see.
```

Comment:
why was this needed? I'm just curious
also there's a decent bit of space wasted here overall (7 lines for one command, unless we have plans to expand it in the future at some point)

Reply:
It's not a super elegant solution, but it's tough to find a lot of information for Windows from the GUI (such as program versions). We could definitely remove this functionality and just include a posh command near the checks that need it.